tensorflow/core/platform/env.cc

```diff
@@ -266,6 +270,13 @@ string Env::GetExecutablePath() {
   char unresolved_path[buffer_size];
   _NSGetExecutablePath(unresolved_path, &buffer_size);
   CHECK(realpath(unresolved_path, exe_path));
+#elif defined(__FreeBSD__)
+  int mib[4] = {CTL_KERN, KERN_PROC, KERN_PROC_PATHNAME, -1};
+  size_t exe_path_size = PATH_MAX;
+
+  if (sysctl(mib, 4, exe_path, &exe_path_size, NULL, 0) != 0) {
+    return "Resolving ", exe_path, " failed";
```

Comment:
Looking at this again, this string can easily be interpreted as a path.
Let's leave a comment saying "Resolution of the path failed."
and return ""

Sorry for the change of mind here.